### PR TITLE
DS-6 docstrings

### PR DESCRIPTION
**Pull request by evanfields**
This short PR gives exported functions proper Documenter.jl-ready docstrings. Other strictly internal functions don't need such verbosity.

### CODE REVIEW

File: src/GeohashHilbert.jl [hunk 1]
```diff
@@ -299,7 +331,13 @@ function neighbours(geohash, bits_per_char)
 
 end
 
-function rectangle(geohash, bits_per_char)
+"""
+    rectangle(geohash, bits_per_char = 2)
+
+Return a GeoJSON Dict which encodes as a Feature the rectangle associated with a given
```
Comment (jdreben) [inline src/GeohashHilbert.jl]: that* (instead of which), as there is not leading comma

File: src/GeohashHilbert.jl [hunk 2]
```diff
@@ -261,7 +285,15 @@ function cell_size_deg(precision, bits_per_char = 2)
     return 360 / n, 180 / n
 end
 
-function neighbours(geohash, bits_per_char)
+"""
+    neighbours(geohash, bits_per_char = 2)
+
+Compute the geohashes of the cells neighboring the specified geohash cell. Return a
+dictionary with keys "north", "north-east", "east", etc. and values the geohash string of
```
Comment (jdreben) [inline src/GeohashHilbert.jl]: Nit: Maybe more clear imo to say "keyed by" and "valued by", or have a colon following "keys" and "values". Reads fine as is.

**Review by jdreben — APPROVED** [review]
Looks great!

File: src/GeohashHilbert.jl (hunk 1 above)
Comment (evanfields) [reply src/GeohashHilbert.jl]: Is this a rule? Which only after a preceding comma? I did not know this.

**Pull request merged**